doc/source/rllib.rst

```diff
@@ -0,0 +1,112 @@
+RLLib: Ray's modular and scalable reinforcement learning library
+================================================================
+
+This document describes Ray's reinforcement learning library.
+It currently supports the following algorithms:
+
+-  `Proximal Policy Optimization <https://arxiv.org/abs/1707.06347>`__ which
+   is a proximal variant of `TRPO <https://arxiv.org/abs/1502.05477>`__.
+
+-  Evolution Strategies which is decribed in `this
+   paper <https://arxiv.org/abs/1703.03864>`__. Our implementation
+   borrows code from
+   `here <https://github.com/openai/evolution-strategies-starter>`__.
+
+-  `Deep Q Networks <https://www.cs.toronto.edu/~vmnih/docs/dqn.pdf>`__
+   based on `OpenAI baselines <https://github.com/openai/baselines>`__.
```

Comment:
Should we add this in later when it is more polished?

Reply:
Done